### DISCUSSION TITLE
Where's the obstacle course?

### DISCUSSION THREAD

**Discussion by dscotese** [bisq-network/bisq2#3004] [Q&A] [answered]
The sentence "The UI fields for Bitcoin addressed and transactions do not use validation, so it does not enforce the usage of mainnet Bitcoin." is in https://bisq.wiki/Bisq_Easy. I wanted to make an edit so it says "The UI fields for Bitcoin addresses and transactions do not use validation, so they do not enforce the usage of mainnet Bitcoin." I can't, because I'm not registered in the Wiki.  I hope there is an obstacle course of some kind that allows valuable people (such as I aspire to be) to register and contribute. My next task is to check out the code and see if there's anything I might improve there. Perhaps that IS the obstacle course I seek.

At any rate, if someone who can edit the wiki fixes that sentence, I won't mind a bit.

**Comment by HenrikJannsen** [comment]
@suddenwhipvapor Do you have access to wiki and can help? We actually have now validation but it allows user to override for flexibility.

**Comment by suddenwhipvapor** [comment, marked as answer]
@HenrikJannsen @dscotese 
the article now reads

> The UI fields for Bitcoin addresses and transactions validate the inputs, but users are allowed to override this behaviour for flexibility.

@dscotese wiki write access is granted to older and more trusted contributors, as the wiki is a front-facing knowledge base, and even the most well-intentioned of editors (as I hear has happened in the past) could inadvertently make changes that are either incorrect or formally inappropriate.
Without having to open issues in the Bisq2 GitHub tracker, you can come over to [matrix](https://bisq.chat) and just ask in the Comms room for the edits that you deem necessary. I guess that if you keep at it for long enough and well enough someone will notice and let you have write privileges, but the support team is more than available to make the edits that you suggest, anytime, as long as they are relevant.

**Comment by dscotese** [comment]
Thank you!
